extensions/ql-vscode/src/mocks/mock-gh-api-server.ts

```diff
@@ -1,13 +1,28 @@
+import { MockGitHubApiConfigListener } from '../config';
+
 /**
  * Enables mocking of the GitHub API server via HTTP interception, using msw.
  */
 export class MockGitHubApiServer {
+  private isListening: boolean;
+  private config: MockGitHubApiConfigListener;
```

Comment:
Can this be `readonly` and instantiated inline?

```suggestion
  private readonly config = new MockGitHubApiConfigListener();
```

I'm also happy to leave it like this.

Reply:
I'm not a huge fan of adding logic outside of the constructor (but that runs as part of the constructor).